iphone/Classes/TiUITableViewRowProxy.m

```diff
@@ -528,7 +528,7 @@ - (TiProxy *)parentForBubbling
 
 - (UIView *)view
 {
-  return nil;
+  return rowContainerView;
```

Comment:
@garymathews TiUITableViewRowContainer is derived from UIView. Ideally it should be derived from TiUIView. It may crash at some point because all functions and properties available in TiUIView is not available with TiUITableViewRowContainer.

Reply:
Should `view` enforce `TiUIView`?
```
- (TiUIView *)view
```

Reply:
I've updated the PR, `TiUITableViewRowContainer` is not a subclass of `TiUIView`

Reply:
@garymathews I am not sure if it will work. I guess it may cause layout issues. 1. Can you test  long tableview (at least some tableview rows goes out of screen) with different contents (to verify reusability of cells). 
2. Need to test a bit more around TableViewRow properties.

Reply:
From my testing, everything works as it should with no layout or event issues. I don't think using `TiUIView` is necessary, but will maintain the change.

```JS
const win = Ti.UI.createWindow();
const tableData = [];

for (let i = 1; i <= 100; i++) {
	const row = Ti.UI.createTableViewRow();
	const label = Ti.UI.createLabel({
		color: 'black',
		text: `Row #${i}`,
		height: 50
	});

	row.add(label);
	tableData.push(row);
}

const tableView = Ti.UI.createTableView({ data: tableData });

tableView.addEventListener('click', e => {
	const row = tableData[e.index];
	const message = `Clicked row ${e.index + 1}\n\nrect.width: ${row.rect.width}\nrect.height: ${row.rect.height}`;

	// NOTE: `borderRadius` on TableViewRow has no effect on iOS.
	row.borderRadius = 20;
	row.backgroundColor = 'red';

	Ti.API.info(message);
	alert(message);
});
tableView.addEventListener('longpress', e => {
	const message = `Long-pressed row ${e.index + 1}`;

	Ti.API.info(message);
	alert(message);
});

const restartButton = Ti.UI.createButton({
	title: 'Restart',
	bottom: 5,
	right: 5
});
restartButton.addEventListener('click', _ => {
	Ti.App._restart();
});

win.add([ tableView, restartButton ]);
win.open();
```